dashboard/src/main/java/org/openedx/learn/presentation/LearnFragment.kt

```diff
@@ -164,6 +174,25 @@ private fun Title(
             color = MaterialTheme.appColors.textDark,
             style = MaterialTheme.appTypography.headlineBold
         )
+        IconButton(
```

Comment:
We need to put this behind the config flag.

Reply:
done.